project/dependencies.scala

```diff
@@ -78,16 +80,25 @@ object dependencies extends AutoPlugin {
     )
   }
 
+  private val tapir = Def.setting {
+    Seq(
+      "com.softwaremill.sttp.tapir" %% "tapir-core"               % V.tapir   % Provided,
+      "com.softwaremill.sttp.tapir" %% "tapir-openapi-docs"       % "0.16.16" % Test,
+      "com.softwaremill.sttp.tapir" %% "tapir-openapi-circe-yaml" % "0.16.16" % Test
```

Comment:
```suggestion
      "com.softwaremill.sttp.tapir" %% "tapir-openapi-docs"       % V.tapir % Test,
      "com.softwaremill.sttp.tapir" %% "tapir-openapi-circe-yaml" % V.tapir % Test
```

Reply:
For tests we use the latest version, so Scala Steward updates it and we can ensure that the integration works with the latest version (if something breaks we get a failed PR)